cli/code_scanner.py

```diff
@@ -197,26 +225,34 @@ def pre_receive_scan(context: click.Context, ignored_args: List[str]):
 
 
 def scan_sca_pre_commit(context: click.Context):
+    report = context.obj["report"]
+    scan_parameters = get_default_scan_parameters(None, report)
     git_head_documents, pre_committed_documents = get_pre_commit_modified_documents()
     git_head_documents = exclude_irrelevant_documents_to_scan(context, git_head_documents)
     pre_committed_documents = exclude_irrelevant_documents_to_scan(context, pre_committed_documents)
     sca_code_scanner.perform_pre_hook_range_scan_actions(git_head_documents, pre_committed_documents)
     return scan_commit_range_documents(context, git_head_documents, pre_committed_documents,
-                                       configuration_manager.get_sca_pre_commit_timeout_in_seconds())
+                                       configuration_manager.get_sca_pre_commit_timeout_in_seconds(),
+                                       scan_parameters=scan_parameters)
 
 
 def scan_sca_commit_range(context: click.Context, path: str, commit_range: str):
+    report = context.obj["report"]
+    scan_parameters = get_scan_parameters(path, None, report)
```

Comment:
why None, its boolean

Reply:
since the only use of report is here and his use is just to pass it to another func, pass here the context and get the report value inside